port/nordic/nrf5x/src/hal/drivers.zig

```diff
@@ -0,0 +1,111 @@
+//!
```

Comment:
This file is literally identical to the rp2xxx one! That's the benefit of copying its interface.